exporter/containerimage/export.go

```diff
@@ -274,8 +274,14 @@ func (e *imageExporterInstance) Export(ctx context.Context, src *exporter.Source
 				tagDone(nil)
 
 				if e.unpack {
-					if err := e.unpackImage(ctx, img, src, session.NewGroup(sessionID)); err != nil {
-						return nil, nil, err
+					if opts.RewriteEpoch {
+						// e.unpackImage cannot be used because src ref does not point to the rewritten image
+						err = errors.New("exporter option \"rewrite-epoch\" conflicts with \"unpack\"")
+					} else {
+						err = e.unpackImage(ctx, img, src, session.NewGroup(sessionID))
```

Comment:
Should we change this function so that it takes `Result[Remote]` as parameter. This can be follow-up but I do think we should implement it and not error. Same for push.

Reply:
Sorry, not sure how it relates to this PR

Reply:
There are new conditions now for unpack and push, instead of reusing the code-path. And unpack case errors.